### PR TITLE
switch annotation from sensio/framework-extra-bundle to symfony/routing

### PR DESCRIPTION
**Pull request by GrimpEreau**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Fixed issues |

### CODE REVIEW

**Review by Elorfin — APPROVED** [review]

**Review by chalasr — CHANGES_REQUESTED** [review]
This seems to break tests. No merge please, looking at it.

File: main/core/Controller/APINew/Template/TemplateController.php [hunk 1]
```diff
@@ -138,9 +138,9 @@ public function templatesByLangDeleteBulkAction(Request $request)
      * @EXT\ParamConverter(
      *     "template",
      *     class="ClarolineCoreBundle:Template\Template",
-     *     options={"mapping": {"id": "uuid"}}
+     *     options={"mapping": {"id": "uuid"}},
+     *     methods={"PUT"}
```
Comment (chalasr) [inline main/core/Controller/APINew/Template/TemplateController.php]: This is the culprit! It's not a Route but a ParamConverter :)

File: main/core/Controller/APINew/User/LocationController.php [hunk 2]
```diff
@@ -35,7 +35,7 @@ public function getName()
     use HasOrganizationsTrait;
 
     /**
-     * @Route("/{id}/geolocate", name="apiv2_location_geolocate", methods={"GET"})
+     * @Route("/{id}/geolocate", name="apiv2_location_geolocate@", methods={"GET"})
```
Comment (Elorfin) [inline main/core/Controller/APINew/User/LocationController.php]: I think this is a typo.

Comment (chalasr) [reply main/core/Controller/APINew/User/LocationController.php]: Good catch, fixed. thanks

**Review by chalasr — APPROVED** [review]

**Pull request merged**